NuGet.config

```diff
@@ -12,6 +12,8 @@
     <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <!-- Used for BenchmarkDotNet prerelease packages -->
+    <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
```

Comment:
Icky to need another feed for a single version of a single package. @wtgodbe and @mmitche could we avoid this feed and (ideally) `vssdk` and `vs-impl` too❔

Or, @NTaylorMullen, are you adding this expecting to need frequent BenchmarkDotNet updates before the next NuGet update (which should appear in dotnet-public soon after)❔

Reply:
Don't need frequent updates. Only adding this for net6.0 support on BenchmarkDotNet. Also copied from dotnet/aspnetcore  😉 
https://github.com/dotnet/aspnetcore/blob/8d8deb71082080d2e674d3d60dab04e9ea11c2bd/NuGet.config#L17-L18